### PR TITLE
Python pandas example

### PR DESCRIPTION
**Pull request by jarvis01123**
This example shows downloading a DataExchange CSV asset, importing it to a Data Frame and running df.describe() to print statistics on the CSV. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,7 +33,8 @@ Examples of interacting with the AWS Data Exchange API from the data provider si
 
 ### Python
 
-* [download-entitled-assets](subscribers/python/download-entitled-assets.py): Download all assets to which you've subscribed.
+* [download-entitled-assets](subscribers/python/download-entitled-assets/download-entitled-assets.py): Download all assets to which you've subscribed.
```
Comment (dblock) [inline README.md]: These belong in the subscribers section (but appear in the providers section) of the README

File: README.md [hunk 2]
```diff
@@ -33,7 +33,8 @@ Examples of interacting with the AWS Data Exchange API from the data provider si
...
+* [pandas-describe-csv](subscribers/python/pandas-describe-csv/pandas-describe-csv.py): Download a CSV asset by arn, import it into a pandas Data Frame, and describe() the result.
```
Comment (dblock) [inline README.md]: Since there's a README, don't link to the .py file, but to the folder, so that the README shows.

Quote `describe()`.

Comment (dblock) [inline README.md]: Data Frame should be lowercase, but ARN I think should be upper case.

File: subscribers/python/pandas-describe-csv/README.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+* AWSDataExchangeFullAccess
+
+You'll need to set up some access keys for this User, and export them in your local environment:
+
+```
+$ export AWS_ACCESS_KEY_ID=<your-access-key-id>
+$ export AWS_SECRET_ACCESS_KEY=<your-secret-access-key>
+```
+
+Finally, youll have to subscribe to a product on AWS Data Exchange and copy the arn for a CSV asset for this to work.
```
Comment (dblock) [inline subscribers/python/pandas-describe-csv/README.md]: Maybe link to a product that you have used as an example?

File: README.md (hunk 2 above)
Comment (jarvis01123) [reply README.md]: done

File: subscribers/python/pandas-describe-csv/README.md (hunk 3 above)
Comment (jarvis01123) [reply subscribers/python/pandas-describe-csv/README.md]: done

File: README.md (hunk 2 above)
Comment (jarvis01123) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (jarvis01123) [reply README.md]: done

**Comment by dblock** [comment]
Merged, thanks.

**Pull request merged**